test/helpers.test.ts

```diff
@@ -0,0 +1,24 @@
+import {removeParamsFromUrl} from "../src/lib/helpers";
+
+
+test('testRemoveParamsFromUrl()', async () => {
```

Comment:
A suggestion to have 2 of these tests blocks.

One for all the cases `removeParamsFromUrl` mutates the url and one for cases where it is not supposed to.

This will remove the need to have single line comments 

i.e.

```js
test('removeParamsFromUrl() should not remove path information', () => {
...
})
```

Reply:
OK, done